modules/renter.go

```diff
@@ -98,10 +98,10 @@ type HostDBEntry struct {
 	HistoricUptime   time.Duration `json:"historicuptime"`
 	ScanHistory      HostDBScans   `json:"scanhistory"`
 
-	HistoricSuccessfulInteractions uint64 `json:"historicSuccessfulInteractions"`
 	HistoricFailedInteractions     uint64 `json:"historicFailedInteractions"`
-	RecentSuccessfulInteractions   uint64 `json:"recentSuccessfulInteractions"`
+	HistoricSuccessfulInteractions uint64 `json:"historicSuccessfulInteractions"`
 	RecentFailedInteractions       uint64 `json:"recentFailedInteractions"`
+	RecentSuccessfulInteractions   uint64 `json:"recentSuccessfulInteractions"`
```

Comment:
eesh, somehow I missed that these fields are using camelCase instead of our conventional lowercase. As long as these haven't been exposed in the API yet, we could change them without breaking compatibility. Go ignores case when decoding.